nginx.conf

```diff
@@ -253,6 +253,16 @@ http {
             proxy_pass http://fdk-portal:8080;
         }
 
+
+        location ~* /services/.+ {
```

Comment:
Skal det være /api/services for å følge mønsteret til de andre høsterne?

Reply:
Correct me if I'm wrong @valosnah 
/api/blabla er gammelt mønster, skal fjernes etterhvert, men kan jo legge det til

Reply:
La til /api/services

Reply:
Gammelt mønster, men pluss-poeng for nostalgi